proposals/24-fork-and-go-slate-like.md

```diff
@@ -0,0 +1,131 @@
+# Create fork-and-go Slate-like example project that exercises the stack
+
+Authors: @rvagg
+
+Initial PR: https://github.com/protocol/web3-dev-team/pull/24
+
+## Purpose &amp; impact 
+
+ * Internal team education and experience with our full stack and the pain points for web3 app developers
+ * Create an example project that can be easily forked and evolved to create a web3 app that leverages our stack
+
+#### Background &amp; intent
+
+_Describe the desired state of the world after this project? Why does that matter?_
+
+[Slate](https://slate.host/) / [filecoin-project/slate](https://github.com/filecoin-project/slate) provides an excellent example of a user-focused web3 application that uses our stack. It's evolving more as a _product_ in its own right and as the complexity increases, the example-utility for new developers using our stack decreases.
+
+The project aims to build a slim Slate-like example project that exercises our stack in similar ways to Slate but excludes some of the complexity that is not required to demonstrate the key interactions with our stack.
+
+The project will have to contend with various choices and compromises that web3 developers currently have to make to:
+ * deliver compelling and responsive applications for users while attempting to build toward the ideal of fully decentralised services; and
+ * interact with the more prickly parts of our stack as they currently exist (e.g. having to use third-party tools that make it easier, like Textile).
+
+The project **must** be idiomatic to our target audience—developers creating applications for end-users, who are most likely using existing web2 patterns and skills to level up into the web3 world. Therefore, its choice of code dependencies, development and testing style should be as close as possible to typical for this audience and not include quirky tooling, style or architectural choices that distract from the learning experience.
+
+The project should include documentation that:
+ * walks developers through setting up and running the application,
+ * clearly describes the architecture; and
+ * clearly explains the integration with the web3 stack and how such integration can be achieved by developers for their own applications.
+
```

Comment:
```suggestion

The project should also include a ready to go image of all third party software/services needed, such as PostgreSQL, so that users can just focus on learning our stack.
```

In Slate, users also have to setup a Textile hub account. I still need to investigate this, but it might also be important to make this as frictionless as possible.